samples/server/petstore/java-play-framework-fake-endpoints/public/swagger.json

```diff
@@ -749,8 +749,8 @@
           "type" : "array",
           "items" : {
             "type" : "string",
-            "default" : "$",
-            "enum" : [ ">", "$" ]
+            "enum" : [ ">", "$" ],
```

Comment:
theses changes could be ignored, they are caused by the core that doesn't do some sorting and because of this, the order is totally random.